src/components/seo.js

```diff
@@ -66,6 +72,14 @@ function SEO({ description, lang, meta, title }) {
           name: `twitter:description`,
           content: metaDescription,
         },
+        {
+          name: `twitter:image`,
+          content: `${site.siteMetadata.url}${site.siteMetadata.icon}`,
+        },
+        {
+          name: `og:image`,
+          content: `${site.siteMetadata.url}${site.siteMetadata.icon}`,
+        },
```

Comment:
technically you don't actually need twitter:image if you have og:image as well as twitter fallsback